core/include/gnuradio-4.0/Settings.hpp

```diff
@@ -409,9 +406,6 @@ class CtxSettings : public SettingsBase {
 
                 // detect whether field has one of the DEFAULT_TAGS signature
                 if constexpr (settings::isWritableMember<Type, MemberType>()) {
-                    if constexpr (std::ranges::find(gr::tag::kDefaultTags, refl::data_member_name<TBlock, kIdx>.view()) != gr::tag::kDefaultTags.cend()) {
-                        _autoForwardParameters.emplace(memberName);
```

Comment:
`_autoForwardParameters` need to be initialised with the `gr::tag::kDefaultTags` list, and keys that are being set explicitely via the users removed from the auto-forwarding list to avoid having parameters propagated that are statically set.